cyclonedx_py/parser/pipenv.py

```diff
@@ -43,14 +44,26 @@ def __init__(
         debug_message('loading pipenv_contents')
         pipfile_lock_contents = json.loads(pipenv_contents)
         pipfile_default: Dict[str, Dict[str, Any]] = pipfile_lock_contents.get('default') or {}
+        self._process_items(pipfile_default, 'default', use_purl_bom_ref, debug_message=debug_message)
 
+        if "dev" not in omit_category:
```

Comment:
:x: do not use custom or arbitrary values.

`omit_category` should be taken literally.
the values are the actual ones in that domain "Pipenv".

This is a pipenv-specific implementation.
So lets generate a enum that holds well-known values for pipfile's "category".
and lets `omit_category` be a list of values in that Enum.
The categories are probably the well-known values mentioned in https://github.com/CycloneDX/cyclonedx-property-taxonomy/blob/ead9513cea439e459b675b6c505854ffb1ad30f7/cdx/pipenv.md

Reply:
i will refactor/rework this after merge.